tools/marvin/marvin/lib/base.py

```diff
@@ -3377,6 +3377,40 @@ def list(cls, apiclient, **kwargs):
             cmd.listall = True
         return(apiclient.listPortableIpRanges(cmd))
 
+
+class NuageUnderlayPublicIpRange:
```

Comment:
@rhtyd I am convinced with your reasoning, I will move the module to our directory plugins/nuagevsp, and will update you :)